desktop/cmp/grid/impl/filter/ColumnHeaderFilterModel.js

```diff
@@ -74,7 +78,7 @@ export class ColumnHeaderFilterModel extends HoistModel {
         this.column = column;
         this.fieldSpec = filterModel.getFieldSpec(column.field);
 
-        this.valuesTabModel = new ValuesTabModel(this);
+        this.valuesTabModel = this.enableValues ? new ValuesTabModel(this) : null;
```

Comment:
seems like this could be the only place we reference enableValues?   And just don't include the tab in the container.  Then null checks everywhere else on the valuesTabModel, and the activeTab can never be 'valuesFilter'